src/BootstrapMixin.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import styleMaps from './styleMaps';
-import CustomPropTypes from './utils/CustomPropTypes';
+import { keyOf } from 'react-prop-types';
```

Comment:
In an effort to keep things lean should this be using `import keyOf from 'react-prop-types/lib/keyOf';` instead?